storage/redis/builders.go

```diff
@@ -0,0 +1,21 @@
+package redis
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/lovoo/goka/storage"
+
+	redis "gopkg.in/redis.v5"
+)
+
+// RedisBuilder builds redis storage.
+func RedisBuilder(client *redis.Client, namespace string) storage.Builder {
+	return func(topic string, partition int32) (storage.Storage, error) {
+		if namespace == "" {
+			namespace = strconv.FormatInt(time.Now().UnixNano(), 10)
```

Comment:
It would be better to return an error here and make the namespace a required parameter. Using the time as the namespace would mean that the application has to recover every launch and leave a lot of obsolete data from previous runs in Redis.